.changeset/calm-apes-collect.md

```diff
@@ -0,0 +1,7 @@
+---
+'@firebase/webchannel-wrapper': minor
```

Comment:
@Feiyang1 Do we also need to explicitly add '@firebase/firestore' and 'firebase'? i assume this is needed to make sure they get released, but wanted to double check.

Reply:
No. It will trigger `patch` release for `@firebase/firestore` and `firebase` automatically which I guess is what you want.

Reply:
Why is it a minor release? looks like a patch release to me.

Reply:
Note that there is 5% increase in the library size. We didn't upgrade closure to the latest yet exactly because of it. @hsubox76 investigated it and found it's because the usage of Set is no longer stripped in the closure library - https://github.com/google/closure-library/issues/1070. However the closure team has not done anything to fix it.

Reply:
We settled on a "minor" release to mitigate some of the risks associated with jumping between major versions in google-closure-library. Have we previously just released patch releases for these updates?

Reply:
Yes, we have done patch releases for these updates in the past. As long as the Firestore node tests pass, we should be confident that it works, right?

In case you actually want to bump minor for `@firebase/firestore` and thus `firebase`, you should include them explicitly in the changeset.